src/AutoMerge/github.jl

```diff
@@ -112,12 +112,7 @@ end
 
 function get_changed_filenames(api::GitHub.GitHubAPI, repo::GitHub.Repo, pull_request::GitHub.PullRequest; auth::GitHub.Authorization)
     files = GitHub.pull_request_files(api, repo, pull_request; auth=auth)
-    n = length(files)
-    filenames = Vector{String}(undef, n)
-    for i = 1:n
-        filenames[i] = files[i].filename
-    end
-    return filenames
+    return [file.filename for file in files]
```

Comment:
Or `return getfield.(files, :filename)` if that would be preferred.

Reply:
I think we should stick with `getproperty` as the current code uses; for the comprehension vs broadcasting, I think the comprehension is slightly easier to understand, but either way seems OK.

Reply:
Let's leave it as it is.